hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -318,22 +319,64 @@ public static void initOptions(HugeConfig conf,
                 cf.optimizeUniversalStyleCompaction();
             }
 
-            cf.setNumLevels(conf.get(RocksDBOptions.NUM_LEVELS));
-            cf.setCompactionStyle(CompactionStyle.valueOf(
-                    conf.get(RocksDBOptions.COMPACTION_STYLE)));
+            int numLevels = conf.get(RocksDBOptions.NUM_LEVELS);
+            List<CompressionType> compressions =
+                                  conf.get(RocksDBOptions.COMPRESSION_TYPES);
+            E.checkArgument(compressions.isEmpty() ||
+                            compressions.size() == numLevels,
+                            "Elements number of '%s' must be the same as '%s'" +
```

Comment:
same as num levels

Reply:
This intent has been provided by RocksDBOptions.NUM_LEVELS.name()